includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -443,9 +443,6 @@ public function get_upe_available_payment_methods() {
 		$available_payment_methods = [];
 
 		foreach ( $this->payment_methods as $payment_method ) {
-			if ( ! $payment_method->is_available() ) {
-				continue;
-			}
```

Comment:
I had to get rid of this because prior to this PR every `WC_Stripe_UPE_Payment_Method` class was hardcoded to return `true` from `is_available()`. 

Because I changed that is_available() call to refer to whether the payment method is available on the checkout. That's consistant how payment methods `is_available()` function should work.

Reply:
Since `is_available_for_account_country` is new, isn't it possible that it does not exist here for some reason? (it is not part of the class we are extending)

Reply:
Yeah good question @wjrosa. 

I don't believe there's a risk of that happening.

The `WC_Stripe_UPE_Payment_Gateway` currently isn't extended by anything so there's no risk of a parent class with different `$this->payment_methods` objects falling back to this version of `get_upe_available_payment_methods()` and then erroring because the `is_available_for_account_country()` function doesn't exist on their payment method instances. 

On top of that, the `WC_Stripe_UPE_Payment_Gateway::payment_methods` property currently includes all `WC_Stripe_UPE_Payment_Method` classes and that class has a `is_available_for_account_country()` function.

So with that in mind, I can't see this causing any issues at the moment. It's a fairly simple change to be defensive though so I've pushed https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2820/commits/b60906f1a7cb6f02bf812306e5118fd889dc066e to defend against this risk.

Reply:
Gotcha! Thanks for addressing this anyway